src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,55 @@
+(ns quo2.components.lowest-price
+  (:require [quo.design-system.colors :as colors]
+            [quo.react-native :as rn]
+            [re-frame.core :as re-frame]
+            [status-im.utils.number :as number-util]))
+
+(defn lowest-price
+  [{:keys [top-value
+           bottom-value
+           margin-top
+           top-value-text-color
+           top-value-bg-color
+           bottom-value-bg-color
+           bottom-value-text-color]}]
+  (let [{:keys [height width]} @(re-frame/subscribe [:dimensions/window])]
+    [rn/view {:style {:display :flex
+                      :flex-direction :column
+                      :width "100%"
+                      :margin-top (number-util/numerical-string margin-top)
```

Comment:
i believe `(int margin-top)` should work

Reply:
I tried it
But got
`(int "2")
Execution error (ClassCastException) at user/eval5 (REPL:1).
class java.lang.String cannot be cast to class java.lang.Character (java.lang.String and java.lang.Character are in module java.base of loader 'bootstrap')`

Reply:
mhm that's weird , why java ?

Reply:
Oh
Was running a normal REPL
but on the CLJS REPL it fails too
I'll post here what i got previously

Reply:
works fine for me in cljs

Reply:
You're correct, I tried it in a ClojureScript REPL and it worked .. I'll change it

Reply:
Apologies
I changed it, I was using the normal Java's CLJ REPL to test.
I will use the CLJS one from now on for Status-mobile